test/Dialect/FIRRTL/errors.mlir

```diff
@@ -2394,7 +2406,47 @@ firrtl.circuit "MismatchedCases" {
       portDirections = 0 : i0,
       portNames = [],
       annotations = [],
-      portAnnotations = []
+      portAnnotations = [],
+      layers = []
     } : () -> ()
   }
 }
+
+// -----
+
+firrtl.circuit "Top" {
+  firrtl.layer @A bind {}
+  firrtl.option @O {
+    firrtl.option_case @C
+  }
+  // expected-note @below {{original module declared here}}
+  firrtl.module @Foo() attributes {layers = [@A]} {}
+  firrtl.module @Bar() attributes {layers = [@A]} {}
+  firrtl.module @Top() {
+    // expected-error @below {{'firrtl.instance_choice' op The required layers must be [@A], but got []}}
+    firrtl.instance_choice foo {layers = []} @Foo alternatives @O {
```

Comment:
Nit/question: can the `layers` could be elided here, but leaving them in may have a more explanatory effect?

Reply:
Just for explanation, but I'm removing it.